CHANGELOG/CHANGELOG-6.0.md

```diff
@@ -0,0 +1,74 @@
+# Release notes for v6.0.0 
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v5.0.1
+
+## Breaking Changes
+
+### API Removal
+
+- Cherry-pick 704: Remove VolumeSnapshot v1beta1 APIs and generated functions. Please update to VolumeSnapshot v1 APIs as soon as possible. ([#709](https://github.com/kubernetes-csi/external-snapshotter/pull/709), [@RaunakShah](https://github.com/RaunakShah))
+
+## Changes by Kind
+
+### API Change
+
+- Add SourceVolumeMode field to VolumeSnapshotContents. ([#665](https://github.com/kubernetes-csi/external-snapshotter/pull/665), [@RaunakShah](https://github.com/RaunakShah))
+- Cherry-pick #683: Change SourceVolumeMode type to v1.PersistentVolumeMode. ([#686](https://github.com/kubernetes-csi/external-snapshotter/pull/686), [@RaunakShah](https://github.com/RaunakShah))
+- Update snapshotter module to v6 and client module to v5. ([#670](https://github.com/kubernetes-csi/external-snapshotter/pull/670), [@RaunakShah](https://github.com/RaunakShah))
+- Cherry-pick #673: Upgrade Volume Snapshot client to v6. ([#673](https://github.com/kubernetes-csi/external-snapshotter/pull/673), [@RaunakShah](https://github.com/RaunakShah))
```

Comment:
Do we want to keep the "Cherry-pick ..." part of the PR description? I see it in a lot of the other commits too